vlad.metelap/dup.rb

```diff
@@ -0,0 +1,13 @@
+def dup(array_str)
+  array_str.map! { |str| str.squeeze}
```

Comment:
Please take a look on the task use of #.squeeze method is forbidden

Reply:
task was redone